apps/api/.env.example

```
@@ -46,7 +47,7 @@ CLOUDFLARE_SECRET_ACCESS_KEY=""
 # Without it the cookie endpoints return 503 and HLS playback won't start —
 # uploads still work, so leave it unset for upload-only local development.
 # Generate one with: node -e "console.log(require('crypto').randomBytes(48).toString('base64url'))"
-HLS_SIGNING_SECRET=""
+HLS_SIGNING_SECRET="dev-hls-signing-secret-classroomio"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)